src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -964,43 +964,44 @@ def _match_host_names_from_cert(hostnames_from_cert, hostnames_in_webapp):
 def create_function(resource_group_name, name, storage_account, plan=None,
                     consumption_plan_location=None):
 
-    if plan is not None and consumption_plan_location is not None:
-        raise CLIError("Use Plan or Consumption Plan Location")
+    if ((plan and consumption_plan_location) or
+            (plan is None and consumption_plan_location is None)):
+        raise CLIError("Use either Plan or Consumption Plan Location")
```

Comment:
please use the error Travis suggested in the previous PR `usage error: --plan NAME_OR_ID | --consumption-plan-location LOCATION`